include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
I think it's best to use standard definitions for such constants. We just need to make sure it doesn't break portability; e.g., probably stay away from POSIX, does it work with CUDA, etc.